src/radical/pilot/session.py

```diff
@@ -254,11 +254,32 @@ def _init_primary(self):
                                           url=bcfg['addr_pub'],
                                           log=self._log,
                                           prof=self._prof)
+        ru.zmq.Subscriber(channel=rpc.CONTROL_PUBSUB, url=bcfg['addr_sub'],
```

Comment:
should it be assigned to `self._ctrl_sub` and then stopped (`self._ctrl_sub.stop()`) in the `close` method?

(*) and same for `self._ctrl_pub` - being stopped explicitly?